File: FrontEnd/internal-app/src/app/core/models/table/paging.model.ts
import { AppSetting } from '../../config/app-setting.config';

/**
 * Paging model
 * Use for paging table
 */
export class PagingModel {
  pageSize: number;     // Page size number
  pageIndex: number;    // The current page

  constructor(page?: number) {
    this.pageIndex = page ? page : 1;
    this.pageSize = AppSetting.table.pageSize;
  }
}

File: BackEnd/InternalApplication/Core/Core.Common/Models/PagingModel.cs
namespace Core.Common.Models
{
    /// <summary>
    /// Paging model.
    /// </summary>
    public class PagingModel
    {
        /// <summary>
        /// Gets or sets page size.
        /// </summary>
        public int PageSize { get; set; }

        /// <summary>
        /// Gets or sets page index (Current page).
        /// </summary>
        public int PageIndex { get; set; }
    }
}
